internal/handler/rest/authentication/authenticationhandler/signout_test.go

```diff
@@ -22,6 +22,22 @@ import (
 )
 
 func TestAuthenticationHandler_SignoutStudent(t *testing.T) {
+	/*
+		 	Angus: Look into how to run tests in parallel with t.Parallel(). My rules that almost all tests should be
```

Comment:
Very interesting, I didn't know that. While I have implemented t.Parallel() in certain tests, I hadn't adopted it as a standard approach until now 😄. I think when the application grow it can really make the difference!